### DISCUSSION TITLE
Compatibility

### DISCUSSION THREAD

**Discussion by ALMIGHTY-VC** [Alex313031/Thorium-Android#62] [Q&A]
I use an redmi y3 miui 11 a10 with an octa core 1.8ghz max cpu is my phone too old be focused under optimization made for modern devices by thorium

**Comment by ALMIGHTY-VC** [comment]
If yes which version should i hse though my phone supports arm 64 but i have 3gb ram only

**Reply by gz83** [reply]
You should be able to use any of the versions we've released

**Reply by ALMIGHTY-VC** [reply]
> You should be able to use any of the versions we've released

As far as i know thorium is optimized for modern devices does the modern here include mine too🤩

**Reply by gz83** [reply]
If you can install and open the Thorium browser, it means you can use it normally.

**Reply by ALMIGHTY-VC** [reply]
> If you can install and open the Thorium browser, it means you can use it normally.

Okay will try thanks for leading and helping me

**Reply by ALMIGHTY-VC** [reply]
> If you can install and open the Thorium browser, it means you can use it normally.

It suddenly crashes even though i browsed and opened it  java.lang.UnsatisfiedLinkError: No implementation found for void J.N.MSkFYnu9(long, int, int, int[], int, long) (tried Java_J_N_MSkFYnu9 and Java_J_N_MSkFYnu9__JII_3IIJ)
	at J.N.MSkFYnu9(Native Method)
	at org.chromium.components.safe_browsing.SafeBrowsingApiBridge.startUriLookupBySafeBrowsingApi(chromium-Thorium_Public.apk-stable-636721804:13)
	at Kw.run(chromium-Thorium_Public.apk-stable-636721804:484)
	at java.lang.Thread.run(Thread.java:919)

**Comment by ALMIGHTY-VC** [comment]
Now it doesnt even opens and crashes

**Comment by gz83** [comment]
The Android version no longer crashes.
